BondageClub/Scripts/Drawing.js

```diff
@@ -615,27 +656,26 @@ function DrawInvertImage(Img) {
 function GetWrapTextSize(Text, Width, MaxLine) {
 
 	// Don't bother if it fits on one line
-	if (MainCanvas.measureText(Text).width > Width) {
-		var words = Text.split(' ');
-		var line = '';
-
-		// Find the number of lines
-		var LineCount = 1;
-		for (let n = 0; n < words.length; n++) {
-			var testLine = line + words[n] + ' ';
-			if (MainCanvas.measureText(testLine).width > Width && n > 0) {
-				line = words[n] + ' ';
-				LineCount++;
-			} else line = testLine;
-		}
-
-		// If there's too many lines, we launch the function again with size minus 2
-		if (LineCount > MaxLine) {
-			MainCanvas.font = (parseInt(MainCanvas.font.substring(0, 2)) - 2).toString() + "px arial";
-			return GetWrapTextSize(Text, Width, MaxLine);
-		} else return;
+	if (MainCanvas.measureText(Text).width > Width) return;
```

Comment:
This logic seems inverted to me

```suggestion
	if (MainCanvas.measureText(Text).width <= Width) return;
```